test/node/decimal128_tests.js

```diff
@@ -1174,37 +1174,6 @@ describe('Decimal128', function () {
     done();
   });
 
-  it('Support toBSON and toObject methods for custom mapping', function (done) {
-    // Create a custom object
-    var MyCustomDecimal = function (value) {
-      this.value = value instanceof Decimal128 ? value.toString() : value;
-    };
-
-    MyCustomDecimal.prototype.toBSON = function () {
```

Comment:
what is this?

Reply:
This is the test that demonstrated the `toObject` functionality, it also demonstrated that you can pass in your custom type if it defines a `toBSON` method so you could have a two-way round trip, so the may be a little bloated in scope

Reply:
so should we keep the `toBSON` test or is there another one already?

Reply:
[We're covered here](https://github.com/mongodb/js-bson/blob/NODE-4461-rm-toObject-d128/test/node/to_bson_test.js#L6). `toBSON` is a generic solution for adding a BSON transformation to an object, there isn't a special case for Decimal128, if the method is defined it will be used